### DISCUSSION TITLE
how to impersonate as a technical user for execution

### DISCUSSION THREAD

**Discussion by SprickW** [jhuckaby/Cronicle#649] [Q&A]
I think it should be quite normal to automate apps that devops people run manually using Cronicle.

But what would be the way to accomplish that? I assume some custom plugin is needed, but I fail with my tries. Is there some "cookbook" to follow?

Greetings Wolfgang

**Comment by mikeTWC1984** [comment]
I'd think most of the devops like things can be done via scripts. Can you give some example of such app (that cannot be run via shell plugin)?
